docs/getting_started/index.rst

```diff
@@ -1,115 +1,143 @@
 Getting Started
 ===============
 
-.. note:: This document assumes you have a `working Ansible
+.. Note:: This document assumes you have a `working Ansible
           installation`_. If you don't, install Ansible before
           continuing. This can be done simply by running ``pip install -r
           requirements.txt`` from the root of the project.
 
-The Microservices Infrastructure project uses Ansible to bring up
+          It also assumes you have a working Terraform installation which you can download from `Terraform downloads`_.
+
+General Information about Mantl with Ansible
+============================================
+
+The Mantl project uses Ansible to bring up
 nodes and clusters. This generally means that you need three things:
 
- 1. hosts to use as the base for your cluster
- 2. an `inventory file`_ with the hosts you want to be modified.
- 3. a playbook to show which components should go where.
+1. hosts to use as the base for your cluster
+2. an `inventory file`_ with the hosts you want to be modified. Mantl includes ansible inventory within its `Dynamic inventory for Terraform.py`_.
+3. a playbook to show which components should go where. Mantl organizes its components in `sample.yml`_, which we recommend copying to ``mantl.yml`` for the possibility of later customization. You can read more about `playbooks`_ in the Ansible docs.
 
-Provisioning Cloud Hosts
-------------------------
+Preparing to provision Cloud Hosts
+>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
 
 The playbooks and roles in this project will work on whatever provider
 (or metal) you care to spin up, as long as it can run CentOS 7 or
-equivalent. However, here are some guides to get started on common
-platforms:
+equivalent.
+
+There are several preparatory steps to provisioning the cloud hosts that are common to all providers:
+
+Step 1: SSH and SSL
+-------------------
+
+The first step for provisioning with any platform is `generating ssh-keys`_ and `secure copying`_ both the public and private keys to your host.
+
+   .. code-block:: shell
 
-.. toctree::
-   :maxdepth: 1
+        ssh-keygen -t rsa -f /path/to/project/sshkey -C "sshkey"
+        scp -P port /path/to/project/id_rsa* <user>@<host>:.ssh/
```

Comment:
I've actually never had to do this. Maybe note that Terraform does this automatically?